subt/zmq-subt-x4.json

```diff
@@ -111,7 +113,7 @@
               ["rosmsg.image", "app.image"],
 
               ["rosmsg.depth", "depth2scan.depth"],
-              ["rosmsg.scan", "app.scan360"],
+              ["depth2scan.scan", "app.scan360"],
```

Comment:
Does `depth2scan` even make sense for a drone? A drone is peculiar since the direction of movement does not always equal the direction where the lidar is looking. Imagine a relatively fast movement forward and flying above a large rock. It might still make sense (I am not saying it doesn't)... Have you guys deliberated about it? Do you have any reasoning for go/nogo?

Reply:
The part where we detect vertical obstacles appears to be still useful. If the obstacle is vertical, we need to either fly around it to do something more advanced. If what we see is a mild uphill slope, like flying over a stone, we don't mind it and the drone will eventually adjust its altitude.

The parts that makes less sense is worrying about holes in the ground (resp. steep downhill slopes) and ignoring small obstacles on ground and therefore the parametrization disables these.